### DISCUSSION TITLE
Scrolling doesn't work properly

### DISCUSSION THREAD

**Discussion by gacsphere** [neuefische/questions#309] [Web] [answered]
Hello everybody,

I have a [editCardForm](https://github.com/gacsphere/capstone-project/blob/main/components/EditCardForm.js) component to edit the cards in my cards-list. When you click on a card, the form opens. On a small smartphone screen you need to scroll the form to see the whole content. You can scroll it with the scrollingwheel of the mouse or if you test it on the smartphone, you can scroll it by dragging the form up and down. This works good so far when opening the editCardForm via the cards-list.
Now I have the same editCardForm component integrated via the popups on the map. But when I open it by clicking the edit-button on the map-popup, I can't scroll the form with the scrollingwheel of the mouse. I need to drag the scrollbar, which works only on a desktop computer with the scrollingbar, but not on the smartphone. It's impossible to drag the scrollbar with my tapsie fingers. When I try to drag the form to scroll it (like it works when opened via the cards-list), I can see that the map in the background moves instead but not the form. I have a `onClick={(event) => event.stopPropagation()}`on the form in the editCardFrom component. But this doesn't seem to work properly on the map.

The editCardForm component is the same in the [Cards-list component](https://github.com/gacsphere/capstone-project/blob/main/components/Cards.js) and in the [map component](https://github.com/gacsphere/capstone-project/blob/main/components/Map.js) and in both components it's installed the same way. I don't see the difference. So how can I scroll the editCardForm on the map?

[View Deployment on Vercel](https://batscan.vercel.app/)

**Comment by gacsphere** [comment]
Hier noch das Beispiel:
https://user-images.githubusercontent.com/109087424/200797001-dabd7479-50b6-4fdb-ad89-75efd7673d91.mov

**Comment by F-Kirchhoff** [comment, marked as answer]
The problem was, that you didn't display the editForm in a popup component but also tried to render it inside the map component. The solution is to render the editForm __outside__ of the mapContainer, there you can still check if the tobeEditID is not null for conditional rendering.

**Reply by gacsphere** [reply]
Thank you!

**Reply by gacsphere** [reply]
Hi @F-Kirchhoff, 
today I discovered that this change caused another problem. 

When I open the editCardForm via the cards-list, everything seems ok. E.g. I can delete the card by pressing `delete` on the `editCardForm`.
When I open the editCardForm via the map and press delete, it says the props are not found.
(TypeError: Cannot read properties of undefined (reading 'count'))
  `defaultValue={nestingbox.count}`

With commit `1e6a8662` I have the `EditCardFrom` rendered in the `MapContainer` back again. Two commits before `b2a2465` I have it the way you suggested it, rendering outside the `MapContainer`, but it can't reach the props.

**Reply by gacsphere** [reply]
[Here you can test it](https://batscan-iqzx11g0a-gacsphere.vercel.app/)
When you click on `edit` on a popup of the map and then click `delete` the error occurs.
When you click a card in the card list an then click `delete` it works.

[Link to Pull Request](https://github.com/gacsphere/capstone-project/pull/31)
